gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-bin.zip
```

intellij will normally moan if sources aren't provided and offer to update it for you
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-all.zip
```